shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/API/Validation.hs

```diff
@@ -16,12 +16,11 @@ module Shelley.Spec.Ledger.API.Validation
   )
 where
 
-import Cardano.Prelude (NoUnexpectedThunks (..))
+import Cardano.Prelude (NoUnexpectedThunks (..), identity)
```

Comment:
We prefer to avoid additional imports from `Cardano.Prelude`, since at some point (when `NoUnexpectedThunks` gets split out into its own package) we plan to drop the dependency. Can we just use `id`?